### ISSUE TITLE
Android: Progress Bar values not announced

### ISSUE THREAD

**Issue by amarlette** [facebook/react-native#30845]
## Description
When using a screen reader it does not announce current value on focus. Without this, the progress bars are not usable.

## React Native version:
v0.63

**Comment by blavalla** [comment]
@kacieb , @nadiia , @lunaleaps , this issue was with the <ProgressBarAndroid> component, which is not included in the docs on reactnative.dev, so my guess is it's either deprecated, or not an officially supported component. That being said, it is included in the repo, so we should probably fix it, or delete it since it doesn't really work.

**Comment by lunaleaps** [comment]
Yea it looks deprecated: https://reactnative.dev/docs/progressbarandroid. This is not worth looking into since we're recommending users to use a community package instead -- should we close the issue @amarlette?

**Comment by Simek** [comment]
@lunaleaps `ProgressBarAndroid` component is deprecated since `0.63` (https://github.com/facebook/react-native/commit/f295d7f60843a45bb09fc366e497f512c2bc0046) release. 

You might want to report the issue in the extracted package from core (https://github.com/react-native-progress-view/progress-bar-android). It was a part of RNC, but lately the policy have changed and many packages have been moved to their own organizations.

This is also the reason why we lately switch from recommending single package to RN Directory links:
* https://github.com/facebook/react-native-website/pull/2470
